src/main/java/com/glodblock/github/common/parts/PartFluidLevelEmitter.java

```diff
@@ -132,8 +132,7 @@ private boolean isLevelEmitterOn() {
 
         final boolean flipState = this.getConfigManager().getSetting(Settings.REDSTONE_EMITTER)
                 == RedstoneMode.LOW_SIGNAL;
-        return flipState ? this.reportingValue >= this.lastReportedValue + 1
-                : this.reportingValue < this.lastReportedValue + 1;
+        return flipState == (this.reportingValue >= this.lastReportedValue + 1);
```

Comment:
I'll be honest, these two lines are too clever for me, We should make this more verbose, extra SLOC be damned, something like:
```java
final boolean flipState = this.getConfigManager().getSetting(Settings.REDSTONE_EMITTER);
if (flipState == RedstoneMode.LOW_SIGNAL) {
    return this.reportingValue >= (this.lastReportedValue + 1);
} else {
    return this.reportingValue < (this.lastReportedValue + 1);
}
```
(please check the condition is correct, I haven't tested this)

Reply:
Thank you for bringing attention to this. You're right. I fixed it in https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/133/commits/71048a456d9e41f2ac814c9672006226959f3a41